highlevel/main.py

```diff
@@ -87,25 +107,29 @@ async def _get_container(simulation: bool, stub_lidar: bool,
     i = DependencyContainer()
 
     i.provide('configuration', CONFIG)
-
     i.provide('protobuf_router', ProtobufRouter)
 
-    i.provide('odometry_controller', OdometryController)
-    i.provide('localization_controller', LocalizationController)
+    i.provide('odometry_function', lambda: odometry_arc)
+    i.provide('position_controller', PositionController)
+    i.provide('motor_gateway', MotorGateway)
     i.provide('motion_controller', MotionController)
+    i.provide('trajectory_controller', TrajectoryController)
+
     i.provide('strategy_controller', StrategyController)
     i.provide('symmetry_controller', SymmetryController)
     i.provide('obstacle_controller', ObstacleController)
     i.provide('debug_controller', DebugController)
     i.provide('match_action_controller', MatchActionController)
 
-    i.provide('motor_gateway', MotorGateway)
-
     i.provide('probe', Probe)
     i.provide('event_loop', asyncio.get_event_loop())
 
+    i.provide('simulation_configuration', SIMULATION_CONFIG)
```

Comment:
idk, you wrote that line :eyes: 
It does not really matter if it is provided or not in any case.

Reply:
it does not really make sense to provide simulation dependencies when running in production...
I'd prefer to avoid having simulation stuff running at the cup